### PR TITLE
Add data flow documentation

### PR DESCRIPTION
**Pull request by lpil**
new-books

### CODE REVIEW

File: documentation/data-flow.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+The data flow looks like this:
+
+
+    Prismic -> BadgerBrain -> React.London
+
+
+## Updating content
+
+If we want to update the content of the site we log into Prismic, search for
+the appropriate documentent (the React.London community and the contained
```
Comment (pixelblend) [inline documentation/data-flow.md]: Speeling mistakk (documentent)

File: documentation/data-flow.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+events, etc), make our edits, and then save and publish the documents.
+
+React.London queries BadgerBrain for site data every time a user visits the
+site, so the user will always get the latest content in Prismic.
+
+Andy is spearheading the creation of a content editor's guide. Once more
+complete we shall link it, until then ask him to see it.
```
Comment (pixelblend) [inline documentation/data-flow.md]: Maybe just link it now and warn that it's WIP?

**Comment by pixelblend** [comment]
dat prose 👍

File: documentation/data-flow.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+[Badger Brain][brain] application, which in turn gets its data from a managed
+third party content management system called [Prismic][prismic].
+
+[brain]: https://github.com/redbadger/badger-brain
+[prismic]: https://prismic.io
+
+The data flow looks like this:
+
+
+    Prismic -> BadgerBrain -> React.London
```
Comment (pixelblend) [inline documentation/data-flow.md]: is it worth adding something about where the images come from?

File: documentation/data-flow.md (hunk 1 above)
Comment (lpil) [inline documentation/data-flow.md]: Thanks!

File: documentation/data-flow.md (hunk 2 above)
Comment (lpil) [inline documentation/data-flow.md]: I have no idea where it is.

File: documentation/data-flow.md (hunk 3 above)
Comment (lpil) [inline documentation/data-flow.md]: Good call. I'll do that now.

File: documentation/data-flow.md (hunk 2 above)
Comment (pixelblend) [inline documentation/data-flow.md]: Is it [this](https://andrewbestbier1.gitbooks.io/red-badger-prismic-guide/content/)?

**Pull request merged**